reporting/test/request/database.test.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * WhoTracks.Me
+ * https://whotracks.me/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { IDBFactory } from 'fake-indexeddb';
+import { expect } from 'chai';
+import { deleteDB } from 'idb';
+
+import Database from '../../src/request/database.js';
+
+describe('request/database', function () {
+  let database;
+
+  beforeEach(async function () {
+    globalThis.indexedDB = new IDBFactory();
+
+    if (database) {
+      await database.unload();
+      await deleteDB(database.tableName);
+    }
+    database = new Database();
+    await database.init();
+  });
+
+  afterEach(function () {
+    delete globalThis.indexedDB;
+  });
+
+  describe('#tokens', function () {
+    it('#bulkPut', async function () {
+      const token = {
```

Comment:
I find this test confusing. It has one token, but expects a count of two (took me a bit to understand that 2 comes from the "count" property).

Since the hashes seem to have no meaning in the test, reducing the test to a minimal example and making it clearer that the "count" property is tested by it, could help.

Reply:
that's just a sample token, I haven't look into its value.

Can actually delete this test completely, it was only added to ensure the DB works well in nodejs.